src/serde.rs

```diff
@@ -75,22 +75,28 @@ where
 mod tests {
     const INTEGER_ARRAY_EXAMPLE: [u64; 4] = [1, 2, 3, 4];
     use crate::{
-        Array,
         sizes::{U4, U5},
+        Array,
     };
 
     #[test]
     fn deserialize_integer_array() {
-        let serialized = bincode::serialize(&INTEGER_ARRAY_EXAMPLE).unwrap();
-        let deserialized: Array<u64, U4> = bincode::deserialize(&serialized).unwrap();
+        let serialized =
+            bincode::serde::encode_to_vec(INTEGER_ARRAY_EXAMPLE, bincode::config::standard())
```

Comment:
Add `use bincode::serde::{decode_from_slice, encode_to_vec};` to the tests module to make these tests a bit more compact?